build.sbt

```diff
@@ -780,12 +779,12 @@ lazy val prismAgentServer = project
     Docker / dockerUsername := Some("input-output-hk"),
     Docker / dockerRepository := Some("ghcr.io"),
     dockerExposedPorts := Seq(8080, 8085, 8090),
-    dockerBaseImage := "openjdk:11",
+    dockerBaseImage := "amazoncorretto:21.0.0-alpine3.18",
```

Comment:
So you're going to run with the latest version of Java but you are still compiling with the JDK 11 you didn't change the CI. Which is fine but is that intentional?

Reply:
I'll change CI in this PR as well when testing.